cypress/integration/Editor/slate_block_picker_spec.js

```diff
@@ -8,14 +8,16 @@
 
 import {setToken, visitOptions} from "../../support";
 
+before(() => {
+  setToken();
+  cy.visit('/subject-matter/learning-resource/new', visitOptions);
+  cy.get('[cy="slate-block-picker-menu"]').should('not.be.visible');
+});
+
 describe("can enter both element types SlateBlockPicker and SlateVisualElementPicker and add, remove, open and close them", () => {
 
-  beforeEach(() => {
-    setToken();
+  beforeEach(() => {  
     cy.server({ force404: true });
-    cy.visit('/subject-matter/learning-resource/new', visitOptions);
-
-    cy.get('[cy="slate-block-picker-menu"]').should('not.be.visible');
```

Comment:
Tester litt mindre her da, men er kanskje ikke noe problem?

Reply:
Tester ikkje mindre. Den berre laster ikkje sida på nytt for kvar test og dermed vil testene gå litt fortere.

Reply:
Denne sjekken kjøres en gong før alle testene kjøres.

Reply:
Men vi tester ikke at blockpickeren ikke vises ved reload? Ikke at det er verdt å gjøre det hver gang uansett :smile: